packaging/s3plugin_lib.py

```diff
@@ -21,13 +21,13 @@ def make_arbitrary_file(f_name, f_size, buffer_size=32*1024*1024):
             else:
                 to_write = buffer_size
 
-            current_char = random.randrange(256)
+            current_char = random.randrange(128)
```

Comment:
Is the 256 -> 128 intentional?

Reply:
Yes, I was having issues with string encoding otherwise.

Reply:
Since we are removing the istream.  I'll change this back.